x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -79,7 +79,7 @@ export const defineExplainLogRateSpikesRoute = (
           } else {
             end();
           }
-        }, Math.random() * 1000);
+        }, Math.random() * 250);
```

Comment:
A bit unclear why we are using Math.random() here, would be good to add a comment for this.

Reply:
This is temporary code until we implement the real endpoint for log rate spikes, I added a comment about it here: [fa70297](https://github.com/elastic/kibana/pull/132590/commits/fa70297f36d8ff5bc4f581e5940247642e35bef1)